docs/docs/frequently-asked-questions.md

```diff
@@ -102,8 +95,12 @@ To become a consumer chain use this [checklist](./consumer-development/onboardin
 Currently supported versions:
 
 - Hermes 1.4.1
-- Support for the CCV module was added to the Go [relayer](https://github.com/cosmos/relayer) in v2.2.0 but v2.4.0 has significant performance fixes which makes it the earliest suggested version to use.
```

Comment:
Removed this based on Marius' comment [here](https://informalsystems.slack.com/archives/C01QUK0LM47/p1711617528587369?thread_ts=1711616250.978829&cid=C01QUK0LM47).